google-cloud-clients/google-cloud-spanner/src/test/java/com/google/cloud/spanner/SessionPoolStressTest.java

```diff
@@ -237,7 +245,7 @@ private Exception getFailedError() {
   @Test
   public void stressTest() throws Exception {
     int concurrentThreads = 10;
-    final int numOperationsPerThread = 1000;
+    final int numOperationsPerThread = 500;
```

Comment:
As we are adding an additional parameter value to the test set, we are also increasing the number of test cases. This means that the build time also increases from approx 30 seconds to approx 60 seconds. Reducing this value from 1000 to 500 keeps the build time at approx the same value as prior to this change.